contentcuration/contentcuration/viewsets/audited_special_permissions_license.py

```diff
@@ -0,0 +1,57 @@
+from django_filters.rest_framework import BooleanFilter
+from django_filters.rest_framework import CharFilter
+from django_filters.rest_framework import DjangoFilterBackend
+from django_filters.rest_framework import FilterSet
+from rest_framework.permissions import IsAuthenticated
+
+from contentcuration.viewsets.base import ReadOnlyValuesViewset
+from contentcuration.models import AuditedSpecialPermissionsLicense
+
+
+class AuditedSpecialPermissionsLicenseFilter(FilterSet):
+    """
+    Filter for AuditedSpecialPermissionsLicense viewset.
+    Supports filtering by IDs and distributable status.
+    """
+
+    by_ids = CharFilter(method="filter_by_ids")
```

Comment:
We can use the [UUIDInFilter](https://github.com/taoerman/studio/blob/3170f54cf20e3fd2cffc9ec623aaf8e7d0148a5d/contentcuration/contentcuration/viewsets/common.py#L40) here too!